goldens/public-api/angular/ssr/index.api.md

```diff
@@ -8,8 +8,9 @@ import { EnvironmentProviders } from '@angular/core';
 
 // @public
 export class AngularAppEngine {
```

Comment:
Happy to change the names here.

Reply:
I think I'm fine with `AngularAppEngine`. Sometimes it won't be rendering Angular, but I think it's close enough to work.

Reply:
I mean that methods names :)